contracts/TypedMemView.sol

```diff
@@ -155,25 +175,38 @@ library TypedMemView {
         }
     }
 
-    /// Return the null view
+    /**
+     * @notice      Return the null view.
+     * @return      bytes29 - The null view
+     **/
     function nullView() internal pure returns (bytes29) {
```

Comment:
Noticed some return values don't have variable names. Should they, or does it not really matter?

Additionally, the docs usually read `@return <returnVal> - <description>`.  If the return value doesn't have a name, should the docs read `@return <type> - <description>` or just `@return <description>`?

Reply:
Oh, I just realized why some have a variable name in the return and some don't

Reply:
it doesn't matter in most cases. Naming the return value allocates a stack item behind the scenes. Sometimes this is not necessary